packages/access/graphql/resolvers/AccessGrant.js

```diff
@@ -0,0 +1,40 @@
+const campaignsLib = require('../../lib/campaigns')
+const eventsLib = require('../../lib/events')
+
+const { Roles, transformUser } = require('@orbiting/backend-modules-auth')
+
+module.exports = {
+  campaign: (grant, args, { pgdb }) => campaignsLib.findByGrant(grant, pgdb),
+  grantee: async (grant, args, { user: me, pgdb }) => {
+    const grantee =
+      await pgdb.public.users.findOne({ id: grant.granteeUserId })
+
+    if (!Roles.userIsMeOrInRoles(grantee, me, ['admin', 'supporter'])) {
+      return null
+    }
+
+    return transformUser(grantee)
+  },
+  granteeName: async (grant, args, { user: me, pgdb }) => {
+    const grantee =
+      await pgdb.public.users.findOne({ id: grant.granteeUserId })
+
+    const safeUser = transformUser(grantee)
+
+    return safeUser.name || safeUser.email
```

Comment:
It's currently not possible to buy or claim a membership and not have a name. I'd recommend to not fallback to the email and instead print 'Unbekannt' or 'Unbenannt'.